libs/wingc/src/jsify.rs

```diff
@@ -1820,10 +1816,12 @@ impl<'a> CaptureScanner<'a> {
 		self.visit_scope(self.function_scope);
 	}
 
-	fn consider_reference(&mut self, path: &[&Symbol]) {
+	fn consider_reference(&mut self, path: &Vec<Symbol>) {
 		let fullname = path.iter().map(|s| s.name.clone()).collect_vec().join(".");
 
-		let lookup = self.current_env.lookup_nested(path, Some(self.current_index));
+		let lookup = self
+			.current_env
+			.lookup_nested(&path.iter().collect_vec(), Some(self.current_index));
```

Comment:
`path` already a `&Vec`
```suggestion
			.lookup_nested(path, Some(self.current_index));
```

Reply:
I get:

```
mismatched types
expected reference `&[&ast::Symbol]`
   found reference `&Vec<ast::Symbol>`
```

Reply:
```suggestion
			.lookup_nested(path.as_slice(), Some(self.current_index));
```